src/OpenTelemetry.PersistentStorage.Abstractions/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Changelog
 
-// TODO: update changelog
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
Change this to have a different version from the old package?

Reply:
Was not sure about this. I will update